documentation/v0_2_0/gsg/running_workloads.md

```diff
@@ -80,3 +80,6 @@ status:
   }
 ]
 ```
+
+## Example workload
+we have an example workload that can be used to test the system. check it [here](https://project-flotta.io/flotta/2022/09/04/edge-example-app-sense-the-internet-part-1.html). 
```

Comment:
I'd suggest to reword the sentence in an active form. For example "An example workload to test the system is available here".

In addition: what about the other sample app? Can you add some other words to make this subsection more valuable?